File: logic-fizzbuzz/FizzBuzz.rb
#! /usr/bin/env ruby
(0..100).each do |ii|
  msg = ""
  if ii % 3 == 0
    msg += "Fizz"
  end
  if ii % 5 == 0
    msg += "Buzz"
  end
  puts msg.empty? ? ii : msg
end

File: logic-fizzbuzz/FizzBuzz.php
<?php
for($ii = 0; $ii <= 100; $ii++){
    $msg = "";
    if ($ii % 3 == 0) {
        $msg .= "Fizz";
    }
    if ($ii % 5 == 0) {
        $msg .= "Buzz";
    }
    print (empty($msg) ? $ii : $msg) . "\n";
}
?>